MobileCenter/MobileCenter/Internals/Model/MSCoreLog.h

```diff
@@ -0,0 +1,11 @@
+#import <Foundation/Foundation.h>
+#import "MSAbstractLog.h"
+
+@interface MSCoreLog : MSAbstractLog
```

Comment:
I agree with @jaelim-ms. We shouldn't use "Core" in classnames or in our SDK in general. `MSStartServiceLog` works well for me.